src/basic/log.c

```diff
@@ -106,7 +106,8 @@ bool _log_message_dummy = false; /* Always false */
         } while (false)
 
 static void log_close_console(void) {
-        console_fd = safe_close_above_stdio(console_fd);
+        /* See comment in log_close_journal() */
+        (void) safe_close_above_stdio(TAKE_FD(console_fd));
```

Comment:
`safe_close_above_stdio()` does not return error code, but returns value for invalidating fd.
So, please drop the voidification. We voidify function call only when we ignore the error code.